packages/components/popper/src/content.vue

```diff
@@ -194,6 +205,38 @@ onMounted(() => {
     }
   )
 
+  watch(
```

Comment:
This watcher is always triggered when a ElTooltip/ElPopover/ElPopconfirm/etc is shown because it's inside onMounted and it's `{immediate: true}`. The `props.triggerTargetEl` is always `undefined` (except for ElDropdown). 
This ends up removing any `role`, `aria-label`, `aria-modal` and `id` attributes for those components.
I believe this is a regression and i'm not quite sure if there's easy way to fix it.